File: ADS 2024.1/POO/atividades_enviar/exercicio3/q06.java
public static void exibir(String... messages) {
    for (String message : messages) {
        System.out.println(message);
    }
}

public static void main(String[] args) {
    exibir("a", "b");
    exibir("a", "b", "c");
    exibir("a", "b", "c", "d");
}

File: ADS 2024.1/POO/atividades_enviar/exercicio3/q06.js
function exibir() {
    var messages = [];
    for (var _i = 0; _i < arguments.length; _i++) {
        messages[_i] = arguments[_i];
    }
    messages.forEach(function (message) {
        console.log(message);
    });
}
exibir("a", "b");
exibir("a", "b", "c");
exibir("a", "b", "c", "d");
